screwdriver/api/api.go

```diff
@@ -55,7 +56,9 @@ func newClient(sdAPI, sdToken string) *client {
 	return &client{
 		baseURL: sdAPI,
 		jwt:     sdToken,
-		client:  &http.Client{Timeout: timeoutSec * time.Second},
+		client: &retryhttp.Client{
```

Comment:
is the retry count configurable ? What is the default ?

Reply:
Yes, the retry is configurable https://github.com/hashicorp/go-retryablehttp/blob/master/client.go#L331  and defaults to 4.

https://github.com/hashicorp/go-retryablehttp/blob/master/client.go#L49